src/web/shared/Styles.ts

```diff
@@ -207,8 +209,8 @@ export function hubCommsCircleStyle(feature: Feature<Point>) {
     function getCircleStyle(center: Coordinate, radius: number, color: string, lineWidth: number) {
```

Comment:
Can we add the a comment before this function like the driftspeedtobinindex function  /***/